fabtests/include/shared.h

```diff
@@ -260,6 +261,7 @@ extern int listen_sock;
 #define HMEM_OPTS "D:i:H"
 #define INFO_OPTS FAB_OPTS HMEM_OPTS "e:M:"
 #define CS_OPTS ADDR_OPTS "I:QS:mc:t:w:l"
+#define LONG_OPTS "1:"
```

Comment:
Long option can exist w/o matching short version. Using '-1 xxx' for the '--pin-core' option doesn't help much. The LONG_OPTS macro is not needed.

On the other hand, I would be help for to define a enum type for the long option `op` result, such as 
```
enum {
    LONG_OPT_PIN_CORE = 1,
};
```
and use the enum value in define the `long_opts` array and in `ft_parse_long_opts()`. That would help future expansion.

Reply:
Thanks for your comments, @j-xiong. I updated in new rev.